doc/source/jupyterhub/customizing/user-resources.md

```diff
@@ -202,3 +202,40 @@ az aks show --name <YOUR-CLUSTER-NAME> --resource-group <YOUR-RESOURCE-GROUP>
 
 It may take some time for the new cluster nodes to be ready.
 You can use `kubectl get node` to report the current `Ready/NotReady` status of all nodes in the cluster.
+
+### Amazon Web Services Elastic Kubernetes Service (EKS)
```

Comment:
Was this section added by mistake, perhaps by creating a branch based on an outdated version of the repo?

Note that we are using `main` rather than `master` now if you want the latest version of the repo.

Reply:
If you want to add this, please submit an additional PR as it is so clearly distinct and unrelated to the culling docs PR.

Reply:
Oops. Yes, I had already opened a separate PR for this and did not mean for it to be included here. I'll clean it up.